src/modules/selects/MultiSelect/MultiSelect.js

```diff
@@ -1,25 +1,14 @@
-import React, { Component } from 'react'
+import React, { PureComponent } from 'react'
```

Comment:
Good job with using `PureComponent` instead of just `Component` 👍 
But in this case, as far as this is stateless component now - we don't need this to be class-based component anymore.